src/Surfnet/Stepup/Identity/Event/GssfPossessionProvenEvent.php

```diff
@@ -30,10 +30,19 @@
 use Surfnet\Stepup\Identity\Value\StepupProvider;
 use Surfnet\StepupBundle\Value\SecondFactorType;
 use Surfnet\StepupMiddleware\CommandHandlingBundle\SensitiveData\Forgettable;
+use Surfnet\StepupMiddleware\CommandHandlingBundle\SensitiveData\RightToObtainDataInterface;
 use Surfnet\StepupMiddleware\CommandHandlingBundle\SensitiveData\SensitiveData;
 
-class GssfPossessionProvenEvent extends IdentityEvent implements Forgettable
+class GssfPossessionProvenEvent extends IdentityEvent implements Forgettable, RightToObtainDataInterface
 {
+    protected static $whitelist = [
+        'identity_id',
+        'identity_institution',
+        'second_factor_id',
+        'stepup_provider',
+        'preferred_locale',
+    ];
```

Comment:
Probabliy missing:
- email verification required
- commonname
- email

Reply:
email verification is not a whitelisted attribute,
added commonName and email